tests/src/Kernel/fixtures/rendering.yml

```diff
@@ -440,6 +440,59 @@
       'div.ecl-container div.ecl-banner__container div.ecl-banner__content div.ecl-banner__title span.ecl-banner__title-text': Banner title
       'div.ecl-container div.ecl-banner__container div.ecl-banner__content p.ecl-banner__description span.ecl-banner__description-text': Banner description
       'a.ecl-link.ecl-link--cta.ecl-link--icon.ecl-banner__link-cta[href="http://example.com"] span.ecl-link__label': Subscribe
+- array:
```

Comment:
Let's update the test coverage for the other cases to assert no video tag in rendered and add more test cases for the video banner such as:

- multiple video sources and/or tracks
- video without one of the parameters (without poster, then without sources, then without tracks)